packages/docs-app/src/examples/datetime-examples/common/momentDate.tsx

```diff
@@ -11,8 +11,13 @@ import moment from "moment";
 import * as React from "react";
 
 const FORMAT = "dddd, LL";
+const FORMAT_TIME = "dddd, LL LT";
 
-export const MomentDate: React.SFC<{ date: Date; format?: string }> = ({ date, format = FORMAT }) => {
+export const MomentDate: React.SFC<{ date: Date; format?: string; withTime?: boolean }> = ({
+    date,
+    withTime = false,
+    format = withTime ? FORMAT_TIME : FORMAT,
```

Comment:
Interesting. Didn't know you could use an earlier destructured value in the default value of another.

Reply:
oh heck yeah you can! order matters.